R/DataClasses.R

```diff
@@ -523,9 +523,9 @@ setClass("ProcessingStep",
 #'     example, a \code{Chromatogram} for a SRM transition 273 -> 153 will have
 #'     a \code{@precursorMz = c(273, 273)} and a
 #'     \code{@productMz = c(153, 153)}.
-#' 
+#'
```

Comment:
JFTR: first result of the [EditorConfig](https://editorconfig.org) plugin. As I mentioned in #376 these additional spaces at the line ending happened to me all the time (and I always use `git add -p` to exclude them from the commit). In a few commits we would have edited all files and won't be annoyed by trailing whitespaces anymore. Alternatively we could run `for i in *.R; do sed -i 's#[[:space:]]*$##g' ${i}; done` to get rid of them in just one commit.

Reply:
I don't mind if they get fixed successively. I like your editorconfig addon 👍